notebooks/notebooks/iclr2022/elo-vs-visits.ipynb

```diff
@@ -128,7 +128,7 @@
    "outputs": [
```

Comment:
<p>This run is almost finished. There are 169 games left, but for some reason they're taking forever—no games have finished in the past 24 hours. I ran double the games I had intended to so that's contributing this run's long duration.</p><p> </p><p>The Elo curves shouldn't change much, so this is close to the final result.</p><p> </p><p>The red 'X' marking cp103 doesn't fall on the line for cp103. But the red 'X' in our old Elo plot also falls below the line for cp99, so maybe that's nothing new. I don't know whether to trust the red 'X' more or the cp103 line more though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>169 out of how many games in total? I'm a bit worried there's something special about the remaining 169 games that'll mean ignoring them skews our stats, but if it's a small % then it's probably fine.</p>

Reply:
every pairing has finished at least 48 games out of 50, and it's the higher visit bots getting stuck. It's possible that there's a tail of games that are going to end by hitting the move limit? I guess I'll just keep it running for now since the memory usage is only 8GB per GPU and only 1 GPU is having its compute occupied, but I've went and added this Elo plot to the paper since we shouldn't expect the remaining games to change the result much

Reply:
ah man this sounds really annoying :(
If you do end up killing the job, might be worth trying to first gdb in and see where the matches are getting stuck.